app/view.py

```diff
@@ -237,6 +237,7 @@ def post(self):
                 confirmed=False)
             # Write the new NoteWithBadWords to the datastore
             db.put(note)
+            UsageCounter.increment_note_counter(self.repo)
```

Comment:
it seems that this code will be called when you add_note in a view page. so i will leave it for now.